### ISSUE TITLE
How to deal with the imbalance data problem?

### ISSUE THREAD

**Issue by heinze007** [jiegzhan/multi-class-text-classification-cnn-rnn#27]
I tried to reproduce the code, finding out that the classifier would tend to choose the larger classes like THEFT, ASSULT and so forth. How you guys deal with the imbalance data to make them seems more 'balance'?

**Comment by heinze007** [comment]
I've tried to replace the loss function, from Cross Entropy to Weighted Cross Entropy, to give the smaller groups more weights. It works out fairly but the accuracy got only around 70%...
